docs/src/components/Dashboard/components/ComponentList.tsx

```diff
@@ -40,12 +41,12 @@ const Sort = ({
 );
 
 const ComponentList = ({ components, unused }: Props) => {
-  const [data, setData] = React.useState<ComponentData[]>([]);
+  const [data, setData] = React.useState<ComponentData[]>(components);
   const [isSorted, setSorted] = React.useState(false);
-
-  React.useEffect(() => {
-    setData(components);
-  }, [setData, components]);
+  const tailwindStatusData = tailwindMigrationData.reduce((acc, { component, migrated }) => {
+    acc[component] = migrated;
+    return acc;
+  }, {});
```

Comment:
Wouldn't it be easier to store the data in the expected format instead of doing this data manipulation?

```yaml
- Alert: true
- Box: true
...
```

No strong opinion, just a thought.

Reply:
thanks, added